src/_data/sidenav.yml

```diff
@@ -117,10 +116,8 @@
         - title: Widget catalog
           permalink: /development/ui/widgets
     - title: Data & backend
-      permalink: /development/data-and-backend
```

Comment:
Does this have an empty .md file backing it

Reply:
Yes it does! https://docs.flutter.dev/development/data-and-backend